### PR TITLE
Skip publish job checks when image isn't found

### PR DESCRIPTION
**Pull request by dhaiducek**

### CODE REVIEW

File: build/periodic.sh [hunk 1]
```diff
@@ -122,63 +133,65 @@ for repo in $REPOS; do
...
+		# check the prow job history
+		checkProwJob "$repo" "$release" "$HAS_IMAGE"
+		if [ $? -eq 1 ]; then
+			rc=1
+		fi
 	done
 done
 
-cleanup
+# cleanup
```
Comment (ycao56) [inline build/periodic.sh:194]: Is this intended?

**Comment by ycao56** [comment]
Are the submodules intended?

File: build/periodic.sh (hunk 1 above)
Comment (dhaiducek) [reply build/periodic.sh:194]: Nope! 😬

**Comment by dhaiducek** [comment]
Shoot--definitely not. Thanks for catching these! I should have looked at it more closely!

**Review by ycao56 — APPROVED** [review]

**Pull request merged**